modules/network-load-balancer/README.md

```diff
@@ -0,0 +1,26 @@
+# Network Load Balancer Module
+
+<!-- NOTE: We use absolute linking here instead of relative linking, because the terraform registry does not support
+           relative linking correctly.
+-->
+
+This Terraform Module creates a [Network Load Balancer](https://cloud.google.com/load-balancing/docs/network/) using [forwarding rules](https://cloud.google.com/load-balancing/docs/network/#forwarding_rules) and [target pools](https://cloud.google.com/load-balancing/docs/network/#target_pools).
+
+Google Cloud Platform (GCP) Network Load Balancing distributes traffic among VM instances in the same region in a VPC network. 
+
+## Quick Start
+
+* See the [network-load-balancer example](https://github.com/gruntwork-io/terraform-google-load-balancer/tree/master/examples/network-load-balancer) for working sample code.
+* Check out [variables.tf](https://github.com/gruntwork-io/terraform-google-load-balancer/blob/master/modules/network-load-balancer/variables.tf) for all parameters you can set for this module.
+
+## How do you configure this module
+
+This module allows you to configure a number of parameters. For a list of all available variables and their descriptions, see [variables.tf](https://github.com/gruntwork-io/terraform-google-load-balancer/blob/master/modules/network-load-balancer/variables.tf).
```

Comment:
I just realized this is redundant with the second bullet in `Quick Start`, esp seeing it one after the other. Should we just remove this?

Reply:
Yes, I'll remove